Api/spec/models/request_spec.rb

```diff
@@ -1,5 +1,5 @@
-require 'rails_helper'
+# require 'rails_helper'
```

Comment:
Why do you comment all of those lines of code, Don't you think is much better to delete it?